tools/nodejs_api/src_cpp/node_database.h

```diff
@@ -0,0 +1,21 @@
+#ifndef KUZU_NODE_DATABASE_H
+#define KUZU_NODE_DATABASE_H
+
+#include <napi.h>
+#include <iostream>
+#include "main/kuzu.h"
+
+class NodeDatabase : public Napi::ObjectWrap<NodeDatabase> {
```

Comment:
Database class is missing `setLoggingLevel ` but it is a minor feature, so it is OK not to have it for this iteration.